.prettierignore

```diff
@@ -1,6 +1,7 @@
 rerun_cpp/docs/**/*
 
 rerun_py/docs/templates/**
+rerun_py/site/**
```

Comment:
Every time i ran `pixi run fmt` locally prettier would complain about invalid HTML files in here, I decided to exclude it